Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityCameraProfileInspector.cs

```diff
@@ -20,19 +29,33 @@ public class MixedRealityCameraProfileInspector : BaseMixedRealityToolkitConfigu
         private SerializedProperty transparentFarClip;
         private SerializedProperty transparentClearFlags;
         private SerializedProperty transparentBackgroundColor;
-        private SerializedProperty holoLensQualityLevel;
+        private SerializedProperty transparentQualityLevel;
+
+        private static readonly GUIContent addSettingsProviderTitle  = new GUIContent("+ Add Camera Settings Provider", "Add Camera Settings Provider");
+        private static readonly GUIContent removeSettingsProviderTitle = new GUIContent("-", "Remove Camera Settings Provider");
+
+        private static readonly GUIContent componentType = new GUIContent("Type");
+        private static readonly GUIContent supportedPlatformsTitle = new GUIContent("Supported Platform(s)");
 
         private readonly GUIContent nearClipTitle = new GUIContent("Near Clip");
         private readonly GUIContent farClipTitle = new GUIContent("Far Clip");
         private readonly GUIContent clearFlagsTitle = new GUIContent("Clear Flags");
 
-        private const string ProfileTitle = "Camera Settings";
-        private const string ProfileDescription = "The Camera Profile helps configure cross platform camera settings.";
+        private const string profileTitle = "Camera Settings";
```

Comment:
Don't like PascalCase consts? 😜

Reply:
;)

Reply:
to me, private variables are always camel case :)